projects/backend/controllers/appsRendering.ts

```diff
@@ -0,0 +1,33 @@
+import { Request, Response } from 'express'
+import fetch from 'node-fetch'
+
+const fetchRenderedArticle = async (url: string) => {
+    const response = await fetch(url, {
+        headers: {
+            Accept:
+                'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
```

Comment:
Would accepting just `text/html` be enough?

Reply:
it is